node/.prettierignore

```diff
@@ -1,5 +1,6 @@
 # ignore that dir, because there are a lot of files which we don't manage, e.g. json files in cargo crates
 rust-client/*
+*.md
```

Comment:
why?

Reply:
We want the formatter tool to ignore files ending with .md as CHANGELOG.mg file

Reply:
ok ill move it to a new pr